src/bean/functions.cljs

```diff
@@ -1,8 +1,8 @@
 (ns bean.functions)
 
 (defn bean-concat [params]
-  (let [concated-str (reduce (fn [c {:keys [value]}] (str c value))
+  (let [concated-str (reduce (fn [c {:keys [scalar]}] (str c scalar))
```

Comment:
Scalar doesn't apply well here. It usually implies there is also a vector that might have a direction. Couple of suggestions
- `eval-result`
- `ast-result`

Ruled out
- `return-value`
- `val`
- `interpreted-result`

I like `eval-result` since we're calling the ast evaluation `eval-ast`.

Reply:
There are a couple of results as a result of evaluation. At least `:scalar`, `:matrix` and `:error`. Using the term scalar here in the sense of Scalars, Vectors & Matrices.